CIScripts/SpawnTestbed.ps1

```diff
@@ -0,0 +1,55 @@
+if ($Env:ENABLE_TRACE -eq $true) {
+    Set-PSDebug -Trace 1
+}
+
+# Refresh Path and PSModulePath
+$Env:PSModulePath = [System.Environment]::GetEnvironmentVariable("PSModulePath", "Machine")
+$Env:Path = [System.Environment]::GetEnvironmentVariable("Path", "Machine")
+
+# Stop script on error
+$ErrorActionPreference = "Stop"
+
+# Sourcing New-TestbedVMs function
+. $PSScriptRoot\SpawnVM.ps1
```

Comment:
It contains definition of VIServerAccessData class (used 2 lines lower), so I decided to put it before definition of all variables.